### PR TITLE
[#4321] improvement(docs): Update docs for multiple authentication

### PR DESCRIPTION
**Pull request by lw-yang**
### What changes were proposed in this pull request?

Update docs for multiple authentication

### Why are the changes needed?

Fix: #4321

### CODE REVIEW

File: docs/security.md [hunk 1]
```diff
@@ -88,17 +88,18 @@ The URI must use the hostname of server instead of IP.
...
+| Configuration item                                | Description                                                                                                                          | Default value     | Required                                   | Since version |
+|---------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------|-------------------|--------------------------------------------|---------------|
+| `gravitino.authenticator`                         | It is deprecated since Gravitino 0.6.0. Please use `gravitino.authenticators` instead.                                               | `simple`          | No                                         | 0.3.0         |
+| `gravitino.authenticators`                        | The authenticators which Gravitino uses, setting as `simple`,`oauth` or `kerberos`. Multiple authenticators are separated by commas. | `simple`          | No                                         | 0.6.0         |
```
Comment (jerqi) [inline docs/security.md:94]: You need to add the document to explain how resolve the header conflicts. For example, if two authenticators use the basic header.

Comment (lw-yang) [reply docs/security.md]: done. please help review it again

**Review by jerqi — APPROVED** [review]
LGTM.

**Comment by jerryshao** [comment]
@yuqi1129 please also take a look.

File: docs/security.md (hunk 1 above)
Comment (yuqi1129) [reply docs/security.md]: @lw-yang 
Has this comment been fixed?

**Review by yuqi1129 — APPROVED** [review]
LGTM

**Pull request merged**